apps/rush/src/actions/UnlinkAction.ts

```diff
@@ -20,8 +20,9 @@ export default class UnlinkAction extends CommandLineAction {
     super({
       actionVerb: 'unlink',
       summary: 'Delete node_modules symlinks for all projects',
-      documentation: 'This removes the symlinks created by the "rush link" command,'
-        + ' allowing you to resume use NPM commands instead of Rush.'
+      documentation: 'This removes the symlinks created by the "rush link" command. This is useful for'
+       + ' cleaning a repo using "git clean" without accidentally deleting source files, or for using standard npm'
```

Comment:
Also, is this a real concern?  You're implying that "rm -f" is going to follow symlinks and delete your local projects?  I seriously doubt that.  If "gulp clean" does that, we have a major bug in our tools code.

---
In reply to: [116345448](https://github.com/Microsoft/web-build-tools/pull/186#discussion_r116345448) [](ancestors = 116345448)